apps/site/src/views/Home/sections/FAQ/FAQ.module.scss

```diff
@@ -0,0 +1,121 @@
+@use "zothacks-theme" as theme;
+@use "bootstrap-utils" as utils;
+
+.faq {
+	position: relative;
+	display: flex;
+	flex-direction: column;
```

Comment:
Suggestion: don't think flex display makes a difference here? If needed, top padding can be included instead of specifying `margin-top` on the title h2.

Reply:
Hi Taesungh, removing display: flex causes the margin to the top to stop working. If I specify padding from the .faq it causes the lines on the paper to displace. Please let me know if you have a better way to structure the css to avoid this. Thanks!